travis_ci/SETUP.md

```diff
@@ -1,38 +1,43 @@
 # Travis CI FOSSA Setup Instructions
 
 ## Necessary Steps
-  - Follow instructions to [Obtain a FOSSA API Key](/OBTAINING_API_KEY.md)
-  - Request a Github Admin (see #github-admins Slack Channel) to add the FOSSA API Key to your Travis Project
-    * __*NOTE*__ : Make sure to send the API Key only via Encrypted Email
-  - Open the Project's '.travis.yml' file for editing on your Local Machine
-  - Add the following line to the Travis YML 'install' stage exactly as it appears below :  
+
+- Follow the instructions to [obtain a FOSSA API key](/OBTAINING_API_KEY.md)
```

Comment:
also as the same time I improved the copy and formatting, no need for instance to indent all the lists etc.